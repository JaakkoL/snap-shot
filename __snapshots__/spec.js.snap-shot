exports[`compares objects 1`] = {
  "inner": {
    "a": 10,
    "b": 20
  },
  "foo": "foo"
}

exports[`should not store undefined value 1`] = 42

exports[`jest test 1`] = "jest value"

exports[`is 42 1`] = 42

exports[`should be 42 1`] = 42

exports[`should be 80 1`] = 80

