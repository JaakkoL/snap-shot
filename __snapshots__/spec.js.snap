module.exports[`compares objects 0`] = {
  "inner": {
    "a": 10,
    "b": 20
  },
  "foo": "foo"
}

module.exports[`should not store undefined value 0`] = 42

module.exports[`jest test 0`] = "jest value"

module.exports[`is 42 0`] = 42

module.exports[`should be 42 0`] = 42

module.exports[`should be 80 0`] = 80

